docs/features/software-catalog/api.md

```diff
@@ -32,7 +32,7 @@ with a `Bearer` token, which should then be the Backstage token returned by the
 These are the endpoints that deal with reading of entities directly. What it
 exposes are final entities - i.e. the output of all processing and the stitching
 process, not the raw originally ingested entity data. See [The Life of an
-Entity](life-of-an-entity.md) for more details about this process and
+Entity](https://backstage.io/docs/features/software-catalog/life-of-an-entity) for more details about this process and
```

Comment:
You hit an odd one, I've seen this before and I think the correct fix is getting this all on one line

Reply:
For example like this:

![image](https://github.com/backstage/backstage/assets/67169551/5ce252d6-0172-4852-b081-ad34b81bce3a)

Reply:
Ah yes, I will update this PR.